components/display/index.rst

```diff
@@ -147,14 +147,14 @@ Next, create a ``font:`` section in your configuration:
     display:
       # ...
 
-
-If you created a ``fonts`` subfolder, reference the font file as follows:
+You can organise your fonts by using subfolders.
+I.e., if you created a ``_fonts`` subfolder, reference the font file as follows:
```

Comment:
Why not just update the example on line 143?

Reply:
That would assume everyone's using a fonts subfolder?
This folder isn't present by default, so new people might be confused when they don't find the folder.

Reply:
> That would assume everyone's using a fonts subfolder?

No? It is an example, showing it uses a path. Please adjust the above example instead. (there is no reason to have multiple examples to show "how paths work". The goal is to make clear it is a path that is accepted.